MovieDB/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "forst-app",
+  "version": "1.0.0",
+  "description": "My test first app",
+  "main": "index.js",
+  "scripts": {
+    "start": "react-scripts start",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "Vitalii Posidailo",
+  "license": "ISC",
+  "dependencies": {
+    "babel-cli": "^6.26.0",
+    "babel-core": "^6.26.3",
+    "babel-loader": "^8.0.5",
+    "babel-preset-env": "^1.7.0",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-react": "^6.24.1",
+    "express": "^4.16.4",
+    "react": "^16.8.4",
+    "react-dom": "^16.8.4",
+    "react-scripts": "^2.1.8"
+  },
+  "browserslist": [
+    ">0.2%",
+    "not dead",
+    "not ie <= 11",
+    "not op_mini all"
+  ]
+}
```

Comment:
Could you please also add `eslint` to dev-dependencies and setip it with `default`  or `airbnb` rules? Also would be good to add eslint plugin to your IDE.

Reply:
Added. By default use airbnb configurations.